docs/howto/system_testing.md

```diff
@@ -434,6 +434,10 @@ for system tests.
 | skip_transform_validation | boolean |  | Disable or enable the transforms validation performed in system tests. |
 | vars | dictionary |  | Package level variables to set (i.e. declared in `$package_root/manifest.yml`). If not specified the defaults from the manifest are used. |
 | wait_for_data_timeout | duration |  | Amount of time to wait for data to be present in Elasticsearch. Defaults to 10m. |
+| assert.hit_count | integer |  | Exact number of documents to wait for being ingested. |
+| assert.min_count | integer |  | Minimum number of documents to wait for being ingested. |
+| assert.fields_present | []string|  | List of fields that must be present in the documents to stop waiting for new documents. |
+| assert.ingestion_idle_time | duration |  | Minimum time elapsed since the last document was ingested. |
```

Comment:
Do you think any of the new settings (min_count, fields_present or ingestion_idle_time) added here in this PR could be dismissed? are any of those redundant?

Reply:
`ingestion_idle_time` may be redundant in many cases with `hit_count`, and basing tests on timing might also lead to flakiness. I think we can live without this option by now.

Reply:
Removed and updated PR description too.